app/views/two_factor_registrations/prompt.html.slim

```diff
@@ -0,0 +1,14 @@
+.main-content
+  .container
+    .row
+      .sub-panel.login-panel.col-center.col-xs-12.col-sm-10.col-md-8.col-lg-6
+        .col.col-xs-12.col-sm-10.col-md-8.col-lg-10.col-center.text-center
```

Comment:
Seems odd to need both responsive `col` sizes here and on the parent `div`?

Reply:
IIRC this allows the inner content to be narrower than the `.sub-panel` on some bigger screens while filling the `.sub-panel` on smaller screens. I ripped this off from your classes in the `two_factor_authentications` templates here:

https://github.com/brave-intl/publishers/blob/a7ac955d75da1c167fbe362e7f19af5fe195d497/app/views/two_factor_authentications/index.html.slim#L3
https://github.com/brave-intl/publishers/blob/a7ac955d75da1c167fbe362e7f19af5fe195d497/app/views/two_factor_authentications/_totp.html.slim#L5